src/vs/workbench/browser/actions/listCommands.ts

```diff
@@ -494,6 +494,41 @@ function listFocusLast(accessor: ServicesAccessor, options?: { fromFocused: bool
 	}
 }
 
+
+function focusElement(accessor: ServicesAccessor, retainCurrentFocus: boolean): void {
+	const focused = accessor.get(IListService).lastFocusedList;
+
+	// List
+	if (focused instanceof List || focused instanceof PagedList) {
+		const list = focused;
+		list.setSelection(list.getFocus());
```

Comment:
This used to be:

```ts
const list = focused;
list.setSelection(list.getFocus(), fakeKeyboardEvent);
list.open(list.getFocus(), fakeKeyboardEvent);
```

Why this change?

Reply:
Ah damn it, lost this change while rebasing. Updated the PR.